app/src/main/java/org/stepik/android/domain/lesson/model/LessonData.kt

```diff
@@ -17,7 +18,8 @@ data class LessonData(
 
     val stepPosition: Int = 0,
     val discussionId: Long? = null,
-    val discussionThread: String? = null
+    val discussionThread: String? = null,
+    val requiredSection: RequiredSection? = null
```

Comment:
it's not ok to add new field here as it not complete field and exists only in one particular case. It will be better to modify `getLessonDataForDirection` to return new data type with `LessonData` and `RequiredSection`.